test/mainnet/compound-import/compound-import.test.js

```diff
@@ -0,0 +1,124 @@
+const { expect, should } = require("chai");
+const { ethers } = require('hardhat');
+const { buildDSAv2 } = require("../../../scripts/tests/buildDSAv2");
+const { cEthAddress, cDaiAddress, daiAddress, comptrollerAddress } = require("./constants.js");
+const cEthAbi = require("./ABIs/cEthAbi");
+const cDaiAbi = require("./ABIs/cDaiAbi");
+const comptrollerAbi = require("./ABIs/comptrollerAbi");
+const { parseEther, parseUnits } = require("ethers/lib/utils");
+const { encodeSpells } = require("../../../scripts/tests/encodeSpells");
+const encodeFlashcastData = require("../../../scripts/tests/encodeFlashcastData").default;
+
+
+describe('Import Compound', function () {
+    const connectorName = "IMPORT-COMPOUND-TEST-A"
+    let owner; // signers
+    let cEth, cDai, comptroller, Dai; // contracts
+
+    before(async () => {
+        // create (reset) mainnet fork
+        await hre.network.provider.request({
+            method: "hardhat_reset",
+            params: [
+                {
+                    forking: {
+                        jsonRpcUrl: hre.config.networks.hardhat.forking.url,
+                        blockNumber: 13300000,
+                    },
+                },
+            ],
+        });
+
+        // get an account
+        await hre.network.provider.request({
+            method: "hardhat_impersonateAccount",
+            params: ["0x10a25c6886AE02fde87C5561CDD331d941d0771a"],
+        });
+        owner = await ethers.getSigner("0x10a25c6886AE02fde87C5561CDD331d941d0771a");
+
+        await hre.network.provider.send("hardhat_setBalance", [
+            "0x10a25c6886AE02fde87C5561CDD331d941d0771a",
+            parseEther('100000').toHexString()
+        ]);
+
+        // deploy connector contract
+        const contractConnectorFactory = await ethers.getContractFactory("ConnectV2CompoundImport");
+        importCompoundConnector = await contractConnectorFactory.connect(owner).deploy();
+        console.log("Connector address", importCompoundConnector.address);
```

Comment:
Use this for deploying and enabling connector - https://github.com/Instadapp/dsa-connectors/blob/main/scripts/tests/deployAndEnableConnector.ts